tasks/configure_namespace.yml

```diff
@@ -74,6 +74,21 @@
   - source-containers
 
 
+- name: Create configmaps
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'templates/reactor_config_map.yml') | from_yaml }}"
+    validate:
+      fail_on_error: no
```

Comment:
do I read this correctly, that this task won't fail if there is an invalid yaml?

Reply:
Task was changed completely